package.json

```diff
@@ -21,7 +21,7 @@
     "jest": "jest --config ./scripts/jest/config.js",
     "lint": "yarn lint:report --fix",
     "lint:report": "eslint packages scripts",
-    "prerelease": "yarn test && yarn compile && yarn compile:commonjs",
```

Comment:
Was this accidentally removed?

Reply:
No, it was updated to remove the third clause. `yarn compile` already runs all such `compile:` commands. So it seems like a vestige of yore